<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandTeleportObject</class>
 <widget class="QDialog" name="DialogCommandTeleportObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teleport object...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="labelObjectID">
         <property name="text">
          <string>Object ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PanelPrimitiveValue" name="widgetObjectID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>103</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPosition">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>320</height>
        </size>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="radioButtonSelect">
                <property name="text">
                 <string/>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>18</width>
                  <height>41</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PanelSelectPosition" name="panelSelectPosition" native="true">
              <property name="minimumSize">
               <size>
                <width>286</width>
                <height>111</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>111</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,1">
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButtonNumber">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelIDMap">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>ID map:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="PanelPrimitiveValue" name="widgetIdMap" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="3" rowspan="5">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>38</width>
                <height>158</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" rowspan="4">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>128</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="labelX">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="PanelPrimitiveValue" name="widgetX" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="labelY">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="PanelPrimitiveValue" name="widgetY" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="labelYPlus">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Y plus:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="PanelPrimitiveValue" name="widgetYplus" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="labelZ">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="PanelPrimitiveValue" name="widgetZ" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
          <item>
           <widget class="QRadioButton" name="radioButtonObject">
            <property name="text">
             <string>Object (ID):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PanelPrimitiveValue" name="widgetObjectIDPosition" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>27</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>38</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxOptions">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDirection">
            <property name="text">
             <string>Direction:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxDirection">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Unchanged</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>North</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>South</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>West</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>East</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>North-West</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>North-East</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>South-West</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>South-East</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelShadingBefore">
            <property name="text">
             <string>Shading before:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxShadinBefore">
            <item>
             <property name="text">
              <string>ON</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OFF</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelShadingAfter">
            <property name="text">
             <string>Shading after:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBoxShadingAfter">
            <item>
             <property name="text">
              <string>ON</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OFF</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>178</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelSelectPosition</class>
   <extends>QWidget</extends>
   <header>panelselectposition.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandTeleportObject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandTeleportObject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
